themes/default/content/blog/2022-03-10-hierarchical-config/index.md

```diff
@@ -19,7 +19,7 @@ This [issue](https://github.com/pulumi/pulumi/issues/2307) was first open in 201
 - [How to share a config between projects](https://github.com/pulumi/pulumi/issues/5473)
 - [Project-wide variables (not stack specific) #6719](https://github.com/pulumi/pulumi/issues/6719)
 
-This is clearly a feature that our community has asked for and we're currently working on delivering it as soon as we can. However, in the interim, I'm going to show you a few ways that you can get around this now.
+This is clearly a feature that our community has asked for! We're happy to say that [delivered the first part of our plans to support this feature](/blog/project-config-mvp/) in early November 2022. While we believe this new functionality satisfies most customer requests, below are some other approaches you can also use.
```

Comment:
this link text isnt very understandable outside of the context of the rest of the text.

could we work project config in somehow?

```suggestion
This is clearly a feature that our community has asked for! We're happy to say that delivered the first part of our plans to support this feature in early November 2022: [Project config](/blog/project-config-mvp/). While we believe this new functionality satisfies most customer requests, below are other approaches you can use.
```

Reply:
@susanev Let me know if the change in 9eecaf0 looks/sounds better to you.